src/components/filters/filter_icon/filter_icon.ts

```diff
@@ -14,20 +14,10 @@ const CSS = css/* scss */ `
     justify-content: center;
     width: ${FILTER_ICON_EDGE_LENGTH}px;
     height: ${FILTER_ICON_EDGE_LENGTH}px;
-
-    svg {
-      path {
-        fill: ${FILTERS_COLOR};
-      }
-    }
   }
   .o-filter-icon:hover {
     background: ${FILTERS_COLOR};
-    svg {
-      path {
-        fill: white;
-      }
-    }
+    color: white;
```

Comment:
```suggestion
    color: #fff;
```
 let's try to use hex as much as possible :)